src/gait/linear_client.py

```diff
@@ -0,0 +1,124 @@
+from gql import gql, Client
+from gql.transport.requests import RequestsHTTPTransport
+from gql.transport.exceptions import TransportQueryError
+from dotenv import load_dotenv
+import os
+
+class LinearClient:
+    def __init__(self, test_mode: bool = False):
+        self.test_mode = test_mode
+        if not test_mode:
+            load_dotenv(override=True)
+            self._init_client()
+            
+    def _init_client(self):
+        """Initialize Linear GraphQL client"""
+        self.api_key = os.getenv("LINEAR_API_KEY")
+        self.team_id = os.getenv("LINEAR_TEAM_ID")
+        self.project_id = os.getenv("LINEAR_PROJECT_ID")
+        # Check API key and team ID first
+        if not self.api_key:
+            raise ValueError("Missing required Linear API key")
+        
+        if not self.team_id:
+            raise ValueError("Missing required Linear team ID")
+            
+        # Special handling for missing project ID
```

Comment:
For example for this comment, instead of saying special handling for missing project ID, you can say something like: 

Special handling needed as project ID may be set incorrectly. That being said, I'm not sure this actually needs a comment